kratos/utilities/exact_mortar_segmentation_utility.h

```diff
@@ -416,79 +426,79 @@ class KRATOS_API(KRATOS_CORE) ExactMortarIntegrationUtility
-        n[1] = v[0];
-        n[2] = 0.0;
+        normal[0] = -rVector[1];
+        normal[1] = rVector[0];
+        normal[2] = 0.0;
 
-        return n;
+        return normal;
     }
 
     /**
      * @brief This function calculates in 2D the angle between two points
-     * @param PointOrig1 The points from the origin geometry
-     * @param PointOrig2 The points in the destination geometry
-     * @param Axis1 The axis respect the angle is calculated
-     * @param Axis2 The normal to the previous axis
+     * @param rPointOrig1 The points from the origin geometry
+     * @param rPointOrig2 The points in the destination geometry
+     * @param rAxis1 The axis respect the angle is calculated
+     * @param rAxis2 The normal to the previous axis
      * @return angle The angle formed
      */
     static inline double AnglePoints(
-        const PointType& PointOrig1,
-        const PointType& PointOrig2,
-        const array_1d<double, 3>& Axis1,
-        const array_1d<double, 3>& Axis2
+        const PointType& rPointOrig1,
+        const PointType& rPointOrig2,
+        const array_1d<double, 3>& rAxis1,
+        const array_1d<double, 3>& rAxis2
         )
     {
-        array_1d<double, 3> local_edge = PointOrig2.Coordinates() - PointOrig1.Coordinates();
+        array_1d<double, 3> local_edge = rPointOrig2.Coordinates() - rPointOrig1.Coordinates();
         if (norm_2(local_edge) > 0.0)
             local_edge /= norm_2(local_edge);
 
-        const double xi  = inner_prod(Axis1, local_edge);
-        const double eta = inner_prod(Axis2, local_edge);
+        const double xi  = inner_prod(rAxis1, local_edge);
+        const double eta = inner_prod(rAxis2, local_edge);
 
         return (std::atan2(eta, xi));
     }
 
     /**
      * @brief This function checks if two points are the same one
-     * @param PointOrig The points from the origin geometry
-     * @param PointDest The points in the destination geometry
+     * @param rPointOrig The points from the origin geometry
+     * @param rPointDest The points in the destination geometry
      * @return check The check done
      */
     static inline bool CheckPoints(
-        const PointType& PointOrig,
-        const PointType& PointDest
+        const PointType& rPointOrig,
+        const PointType& rPointDest
         )
     {
 //         const double tolerance = std::numeric_limits<double>::epsilon(); // NOTE: Giving some problems, too tight
         const double tolerance = 1.0e-15;
-        return (norm_2(PointDest.Coordinates() - PointOrig.Coordinates()) < tolerance) ? true : false;
+        return (norm_2(rPointDest.Coordinates() - rPointOrig.Coordinates()) < tolerance) ? true : false;
     }
 
     /**
      * @brief This functions calculates the determinant of a 2D triangle (using points) to check if invert the order
-     * @param PointOrig1 First point
-     * @param PointOrig2 Second point
-     * @param PointOrig3 Third point
+     * @param rPointOrig1 First point
```

Comment:
you would make the life of a reviewer much easier if you would do such changes in a separate PR

Reply:
Sorry, you are right